acceptance/lib/puppetserver/acceptance/compat_utils.rb

```diff
@@ -19,7 +25,7 @@ class { 'simmons':
 def rm_vardirs()
   # In order to prevent file caching and ensure agent-master HTTP communication
   # during agent runs we blow away the vardirs, which contains the cached files
-  hosts.each do |host|
+  valid_agents.each do |host|
```

Comment:
@nwolfe Did we have this as `hosts` before because we wanted to blow away the `vardir` on the master as well as all agents?